ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcollection/v1/productcollection/productcollection.html

```diff
@@ -16,6 +16,7 @@
           data-sly-use.paginationTemplate="paginationbar.html"
           data-sly-use.facetSelectorTemplate="facetselector.html">
 
+<div class="results__count">${productCollection.searchResultsSet.totalResults} items</div>
```

Comment:
`items` should be i18n

Also, this line will be rendered if no products are available, but we already have a message for that case.

Reply:
Shouldn't we hide that line if no product is available? It's not useful to display "0 items" if we already show the message that there are no products to display.

Reply:
It also should be singular `1 item` and plural `2 items`.

Reply:
Pushed a change for this